slides/11_Forms.html

```diff
@@ -108,69 +149,62 @@ <h3>Form Properties</h3>
             <section>
-                <h3>Form Bindings</h3>
+                <h3>Forms with baloise design system</h3>
                 <ul>
-                    <li>a simple possibility are two-way bindings with <span class="code">[(ngModel)]</span></li>
-                    <li>It is important to assign a <span class="warn">unique</span> name attribute</li>
+                    <li>
+                        In order to use Baloise layot and style we can use forms from here:
+                        <a href="https://design.baloise.dev/?path=/story/design-form--page" target="_blank">https://design.baloise.dev/?path=/story/design-form--page</a>
+                    </li>
                 </ul>
                 <pre>
-                    <code class="hljs html" data-trim contenteditable>
-                        &lt;form (ngSubmit)="machWas()">
-                            &lt;input type="text" class="form-control"
-                               [(ngModel)]="pizza.name" name="pizzaName">
-                            &lt;button type="submit">Los!&lt;/button>
-                        &lt;/form>
-                    </code>
+                    <code class="hljs html" data-trim><script type="text/template">
+ <form (ngSubmit)="submit()" [formGroup]="form" class="columns is-multiline mt-4">
+  <bal-field class="column is-full py-0" required>
+      <bal-field-label required>Name</bal-field-label>
+      <bal-field-control>
+        <bal-input formControlName="name" placeholder="Enter your name"></bal-input>
+      </bal-field-control>
+    </bal-field>
+</form>></script></code>
                 </pre>
             </section>
             <section>
-                <h3>Task 11.1 - Forms</h3>
+                <h2>Rxjs operators and error handling</h2>
                 <ul>
-                    <!-- TODO REVIEW: is it really 10_Routing_2 ? not 11?-->
-                    <li>Branch: <span class="code">10_Routing_2</span></li>
-                    <li>Create a <span class="code">Visitor</span> component to the right of the table</li>
-                    <li>The component should display the currently selected visitor in a form</li>
-                    <li>Limit yourself to first name and last name, as well as a submit and a delete button</li>
-                    <li>Use the service and REST API to perform save and delete operations </li>
+                    <li>Error handling and handling sucess results is done with rxjs operators in angular. For more info check <a href="https://angular.io/guide/rx-library#error-handling" target="_blank">here</a></li>
+                    <li>Example: Return "response" from the API. If an error happens, return an empty array.</li>
                 </ul>
+                <pre>
+<code class="hljs ts" data-trim>
+this.http.post('/api/data', payload).pipe(
+  map((res: any) => {
+    if (!res.response) {
+      throw new Error('Value expected!');
+    }
+    return res.response;
+  }),
+  catchError(() => of([]))
```

Comment:
we should mention that returning an empty array alone is not really best practice :)

Reply:
I agree we should mention that its always better to show error to user :). This was an example from an official documentation just how to do it. Thanks for comment.